src/pokeblock.c

```diff
@@ -26,8 +26,15 @@
 #include "songs.h"
 #include "safari_zone.h"
 #include "use_pokeblock.h"
+#include "event_data.h"
 #include "pokeblock.h"
 
+extern u8 ewram[];
+
+static EWRAM_DATA u8 gUnknown_02039244 = 0;
+static EWRAM_DATA ALIGNED(4) u8 gUnknown_02039248[4] = {0}; // why aligned to 4 bytes?
```

Comment:
This seems like it would be a struct with four u8 fields, if the code still matches with that.